openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners_assessment.html

```diff
@@ -1,18 +1,32 @@
 {% load i18n %}
 {% spaceless %}
 
-    <div class="staff__grade__form" data-submission-uuid="{{ submission.uuid }}" data-team-submission="{{ teams_enabled|default:"False" }}">
+    <div
+      class="staff__grade__form"
+      data-submission-uuid="{{ submission.uuid }}"
+      data-team-submission="{{ teams_enabled|default:"False" }}"
+      data-team-name="{{ team_name|default:"" }}"
```

Comment:
Just thinking out loud: wondering if it makes sense to have these data tags be conditional, only showing up if team submission is enabled.

Reply:
Actually, you probably don't even need the data flags since we don't programmatically access them in JS.

Reply:
but I do want to unit test the functionality of the template render without querying for the text it uses them for.

Reply:
i.e. verifying that the appropriate variables are sent, and trusting the template to render them appropriately where needed.

Reply:
Fair argument 👍